Formula/zpaq.rb

```diff
@@ -19,7 +19,7 @@ class Zpaq < Formula
   end
 
   resource "test" do
-    url "http://mattmahoney.net/dc/calgarytest2.zpaq"
+    url "https://mattmahoney.net/dc/calgarytest2.zpaq"
     sha256 "ad3b58c245b2a54136d3ff28be78c069b0272eb31f808bf82014134e5913cf7e"
```

Comment:
The checksum of the test file was changed in #66620. I'm not sure why it was changed at that time, but I think we can revert that. In fact, FreeBSD has been using this file since 2012, and the [blame](https://github.com/freebsd/freebsd-ports/blame/f0dadc529fe9c4ee69dd96db92e2d0572d2092aa/archivers/paq/distinfo#L19) suggests that they haven't changed the checksum since.

```suggestion
    sha256 "b110688939477bbe62263faff1ce488872c68c0352aa8e55779346f1bd1ed07e"
```